File: Dashboard Template/vyzor-dotnet/mvc/final/Controllers/TablesController.cs
using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using vyzor.Models;

namespace vyzor.Controllers;

public class TablesController : Controller
{
    private readonly ILogger<TablesController> _logger;

    public TablesController(ILogger<TablesController> logger)
    {
        _logger = logger;
    }

    [Route("/data-tables")]
    public IActionResult data_tables()
    {
        return View("data_tables");
    }

    [Route("/grid-tables")]
    public IActionResult grid_tables()
    {
        return View("grid_tables");
    }

    [Route("/tables")]
    public IActionResult tables()
    {
        return View("tables");
    }

}

File: Dashboard Template/vyzor-symfony/final/src/Controller/TablesController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class TablesController extends AbstractController
{
    
    #[Route('/data-tables', name: 'app_data_tables')]
    public function dataTables(): Response
    {
        return $this->render('tables/data-tables.html.twig', [
            'controller_name' => 'TablesController',
        ]);
    }
    
    #[Route('/grid-tables', name: 'app_grid_tables')]
    public function gridTables(): Response
    {
        return $this->render('tables/grid-tables.html.twig', [
            'controller_name' => 'TablesController',
        ]);
    }

    #[Route('/tables', name: 'app_tables')]
    public function tables(): Response
    {
        return $this->render('tables/tables.html.twig', [
            'controller_name' => 'TablesController',
        ]);
    }

}
